static/images/integrations/logos/ansibletower.svg

```diff
@@ -0,0 +1 @@
667" height="1066.667" viewBox="0 0 800 800"><path d="M384 72c-33.2 3.1-62.4 11.4-91.9 26.1-40 19.8-73.5 49.3-98.8 87.1-8.9 13.4-22.7 41-27.8 55.8-27.8 80.7-14.3 169.4 35.8 236.5C242 532.1 300.6 567 369 577.7c16.3 2.5 57.6 2.5 74-.1 55.9-8.7 103.5-32.9 143-72.5 38.7-38.8 63.3-87.6 71.5-142.1 2.3-14.6 3-44.9 1.6-60.6-5.4-59-31.8-115.1-73.4-156.7C546.8 106.9 497.4 82 443.3 74c-15.9-2.3-45.1-3.3-59.3-2zm31 101c7.7 2.1 6.6-.2 61.5 132 55.6 133.8 54 129.1 50 137.3-3.7 7.8-15.4 11.9-23.5 8.4-1.9-.8-33.6-25.8-70.4-55.6-36.8-29.7-67.2-54.1-67.5-54.1-.3 0-10.3 24.4-22.2 54.3l-21.7 54.2-19.1.3c-10.5.1-19.1.1-19.1-.1 0-.3 25.1-60.9 55.8-134.8 44.4-106.8 56.5-135.1 59-137.6 4.6-4.7 10.5-6.2 17.2-4.3z"/><path d="M393 270c-9 22.5-16.2 41.1-15.9 41.3 2.8 2.2 95.8 75.3 96.7 76 1.1.9 1.1.5 0-2.3-.8-1.9-15.3-37.8-32.3-79.8-17-41.9-31.2-76.2-31.5-76.2-.3 0-8 18.4-17 41zM106.2 635.7c-1.2 1-8.8 18-17.6 39.3l-15.5 37.5 5.6.3c3.1.2 5.9-.1 6.3-.5.4-.4 3.1-7 6.1-14.5 3-7.6 5.8-13.8 6.2-13.8.5 0 9.2 6.7 19.4 15 12 9.7 19.3 15 20.8 15 3 0 5.5-2.5 5.5-5.4 0-3.2-28.8-71.9-30.7-73.4-2.3-1.6-3.7-1.5-6.1.5zm18.1 53.1c2.1 5.1 3.2 5.7-12.7-6.7-7.7-6.1-10.5-8.9-10.1-10 .3-.9 2.2-5.8 4.2-10.9l3.6-9.2 6.7 16.2c3.6 9 7.4 18.2 8.3 20.6zM189.4 635.4c-1.8 1.4-1.9 3.2-2.2 39.5l-.3 38.1H199l.2-28.2.3-28.1 21.5 27.8c11.8 15.3 22.2 28.3 23.1 28.8 2.5 1.6 6.9 0 7.9-2.9.6-1.5 1-18 1-39V635h-12l-.2 28.4-.3 28.4-20-26c-11-14.3-21.2-27.3-22.7-28.9-2.9-3.2-5.4-3.6-8.4-1.5zM317.2 635.4c-8.8 2.2-13.9 7.2-14.9 14.4-1.6 11.8 2.8 16.6 25.8 28.2 9 4.5 17.6 9.4 19.2 10.8 4.7 4.5 3.2 10.2-3.6 13.3-5.2 2.3-19.2 2.5-28.7.3-13.6-3-12.7-3.1-13.9 1.2-.6 2.2-1.1 4.2-1.1 4.6 0 1.3 13.2 4.8 22.5 5.9 14.6 1.7 29-1.5 34.7-7.8 4.5-4.9 6-13.6 3.4-19.7-2.1-5.1-7.6-8.9-26.3-18.1-18.3-9-20.7-11-20.1-16.6.2-2.5 1.1-3.8 3.8-5.4 3-1.7 5.1-2 15-1.9 6.3 0 14.4.6 17.9 1.3 3.5.7 6.7.9 7.2.4.4-.4.9-2.6 1.1-4.8l.3-4-6-1.3c-9.2-2-30-2.4-36.3-.8zM691 635.3c-6.2 1.5-9.4 3-12.8 6.1-7.5 6.7-7.7 19.7-.5 27.4 3 3.2 3.2 3.7 1.6 4.5-10.1 5.7-12.4 21-4.7 31.4 5.6 7.5 22.6 11.4 40.2 9.4 5.3-.7 11.3-1.6 13.3-2.1 3.6-1 3.6-1 2.9-5-.4-2.3-1-4.3-1.3-4.7-.3-.3-3.5 0-7 .6-10.4 1.8-27.5 1.4-32.3-.8-5.7-2.6-8.4-6.4-8.4-11.8 0-3.6.6-5 2.8-7.3 4-3.9 8.8-5 22.8-5 13 0 14.4-.5 14.4-5.2 0-4-2.5-4.8-15.5-4.8-13.3 0-17.5-1.3-20.6-6.4-2.3-3.7-2.4-6.7-.4-10.5 3.1-6.1 15-8.3 33.4-6.2 5.7.7 10.5 1 10.7.8.3-.3.6-2.4.7-4.8.2-4.1.1-4.3-3.3-5.1-6.6-1.5-30.2-1.9-36-.5zM409 674v39h12v-78h-12v39zM474.7 636.8c-1.5 1.7-1.7 5.8-1.7 37.8 0 26.7.3 36.3 1.2 37.2.9.9 7.2 1.2 22.3 1.2 19 0 21.4-.2 25.5-2.1 8.8-4 13.4-10.8 13.4-19.9 0-7.1-3.2-13-9.2-17-3.5-2.3-4.1-3.1-2.8-3.6.9-.3 2.9-2.7 4.4-5.3 2.1-3.6 2.7-5.8 2.6-10.1-.1-6.8-2.3-11.2-7.5-14.9-5.6-4.1-11.5-5.1-30.3-5.1-14.5 0-16.5.2-17.9 1.8zm36.7 9.6c1.8.7 4.3 2.6 5.5 4.2 4.5 5.8.3 15.1-7.8 17.4-2 .5-8.4 1-14.3 1H484v-24h12c8 0 13.2.5 15.4 1.4zm2.7 32.6c8 2.2 12.1 11.2 8 17.7-3.8 6.1-5.9 6.8-22.8 7.2l-15.3.3V678h13.3c7.2 0 14.8.5 16.8 1zM580 672.1c0 23.6.4 37.7 1 39 1 1.8 2.4 1.9 27 1.9h26v-10h-42v-68h-12v37.1z"/></svg>
```

Comment:
About the logo, I think its too small. And I'd recommend cropping it to not include the name "ANSIBLE", since the name is already there as part of our UI.